### PR TITLE
(IAC-858) Release prep 0.5.0

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

File: exe/push-gems.rb [hunk 1]
```diff
@@ -6,27 +6,16 @@
...
-"florin.dragos@puppet.com",
 "daniel.milan@puppet.com",
 "sheena@puppet.com",
 "michael.lombardi@puppet.com",
-"william.hurt@puppet.com",
-"erick@puppet.com",
 "tp@puppet.com",
 "ciaran.mccrisken@puppet.com",
-"sheena@puppet.com",  
+"sheena@puppet.com",
```
Comment (sheenaajay) [inline exe/push-gems.rb:18]:
```suggestion
```

Comment (sheenaajay) [reply exe/push-gems.rb:18]: sorry for the duplicate

Comment (DavidS) [reply exe/push-gems.rb:18]: :-D no worries, I'll get itin . Will also give the list a good sort to reduce confusion down the line.

Comment (sheenaajay) [reply exe/push-gems.rb:16]: :-) perfect

**Pull request merged**